README.rst

```diff
@@ -51,6 +51,14 @@ System requirements:
 - gobject-introspection type libraries for GTK+, Pango
```

Comment:
These are obliquely referring to the gir1.2-... packages.  I forgot to add Soup and GnomeKeyring here (although GnomeKeyring is an optional dependency).

Reply:
There's a semi-undocumented feature where gtimelog can download the list of tasks from a plaintext web page.  I'd been experimenting with using a wiki page for sharing tasks with other team members.  Long story short, it didn't work well and I stopped playing with the feature, but apparently other people still use it, and they send me pull requests like https://github.com/gtimelog/gtimelog/pull/109.

They keyring is for remembering the password if the web page requires authentication.

I'm also planning on implementing [direct SMTP support](https://github.com/gtimelog/gtimelog/pull/129), which may also need to use they keyring to store credentials.  That PR, when finished, will make the keyring mandatory rather than optional.